### PR TITLE
bugfix: fix console jwt log error

### PR DESCRIPTION
**Pull request by jumtp**
- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅱ. Does this pull request fix one issue?

fix: #5324

### CODE REVIEW

**Comment by jumtp** [comment]
完蛋，我加到主分支里面了。需要重新提交吗？

**Review by a364176773 — COMMENTED** [review]
So changed why there is no 401 problem? 401 reason is because the access path is restricted, console-fe/public/ itself is allowed to visit, modify the url reason is because we previously wrote the address is not right to lead to access 404 jump to error and then be intercepted to lead to 401 of this process?
这么改动了为什么就不存在401问题了呢？401的原因是因为访问路径被限制，console-fe/public/本身就是允许访问的，修改url的原因是因为我们之前写的地址不对导致访问404跳转error再被拦截导致401的这个过程吗

**Comment by a364176773** [comment]
> 完蛋，我加到主分支里面了。需要重新提交吗？

bugfix的pr是develop和2.x分支都要提交的

**Comment by jumtp** [comment]
> So changed why there is no 401 problem? 401 reason is because the access path is restricted, console-fe/public/ itself is allowed to visit, modify the url reason is because we previously wrote the address is not right to lead to access 404 jump to error and then be intercepted to lead to 401 of this process? 这么改动了为什么就不存在401问题了呢？401的原因是因为访问路径被限制，console-fe/public/本身就是允许访问的，修改url的原因是因为我们之前写的地址不对导致访问404跳转error再被拦截导致401的这个过程吗

After the seata-console project is packaged, the console-fe/public/ folder does not exist. You can unzip the jar and find that bootstrap.css, console1412.css and other files exist under the css folder

seata-console项目打包后.并不存在console-fe/public/文件夹.你可以解压jar发现.bootstrap.css,console1412.css等文件都存在css文件夹下面


seata-jwt-error1

After you decompress, you can directly modify the static/index file, modify the paths of the .bootstrap.css and console1412.css files to achieve the effect of importing successfully
当你解压过后,可以直接修改 static/index 文件，把.bootstrap.css,console1412.css文件的路径修改一下也可以达到引入成功的效果


seata-jwt-error2

**Comment by jumtp** [comment]
> > 完蛋，我加到主分支里面了。需要重新提交吗？
> 
> bugfix的pr是develop和2.x分支都要提交的

那我现在以同样的方式提交吗，还是说需要我rebase一下

**Comment by slievrly** [comment]
> > > 完蛋，我加到主分支里面了。需要重新提交吗？
> > 
> > 
> > bugfix的pr是develop和2.x分支都要提交的
> 
> 那我现在以同样的方式提交吗，还是说需要我rebase一下

just add change log on：
https://github.com/seata/seata/blob/2.x/changes/en-us/2.0.0.md
https://github.com/seata/seata/blob/2.x/changes/zh-cn/2.0.0.md

File: console/src/main/resources/static/index.html [hunk 1]
```diff
@@ -24,8 +24,8 @@
 	<title>Seata</title>
 
     <link rel="shortcut icon" href="console-fe/public/img/seata_logo_small.jpeg" type="image/x-icon">
```
[the remark this answers is not in the document]
Comment (jumtp) [reply console/src/main/resources/static/index.html]: > shortcut icon也一起改下吧

大意了，已添加~

**Review by liuqiufeng — APPROVED** [review]
LGTM

**Review by liuqiufeng — APPROVED** [review]

**Review by slievrly — APPROVED** [review]
LGTM

**Comment by slievrly** [comment]
I have reproduced the issue.
image

**Comment by slievrly** [comment]
@jumtp thanks for your contribution.

**Pull request merged**